Plugins/Public/playercntl_plugin/MiscCmds.cpp

```diff
@@ -252,6 +263,185 @@ namespace MiscCmds
 		return true;
 	}
 
+	void MiscCmds::LoadListOfReps()
+	{
+		INI_Reader ini;
+
+		string factionpropfile = "..\\data\\initialworld.ini";
+		if (ini.open(factionpropfile.c_str(), false))
+		{
+			while (ini.read_header())
+			{
+				if (ini.is_header("Group"))
+				{
+					uint ids_name;
+					string nickname;
+					while (ini.read_value())
+					{
+						if (ini.is_value("nickname"))
+						{
+							nickname = ini.get_value_string();
+						}
+						else if (ini.is_value("ids_name"))
+						{
+							ids_name = ini.get_value_int(0);
+						}
+
+					}
+					factions[nickname] = ids_name;
+				}
+			}
+			ini.close();
+			ConPrint(L"Resetrep: Loaded %u factions\n", factions.size());
+		}
+	}
+	
+	map<wstring, uint> MiscCmds::Resetrep_Time_limits_load_map(string filename)
+	{
+		INI_Reader ini;
+		string factionpropfile = filename;
```

Comment:
this doesn't seem to be necessary

Reply:
it looks like a function that load 'cooldown restrictions' for certain player folder/account

Reply:
right but why not just either rename `filename` to `factionpropfile`, or drop all references to `factionpropfile` and use `filename`?

Reply:
indeed.